_i18n/nb-no/resources/user-guides/multisig-messaging-system.md

```diff
@@ -0,0 +1,864 @@
+
+Merk for hvor mange handlinger det ikke finnes **noen** dedikert kommando, og `mms next` er den **eneste** måten å la ting gå fremover på. Ikke se etter f.eks. kommandoer for å selektivt behandle visse meldinger på: Hvis det er tid til å behandle noen mottatte meldinger i *venter*-status, vil kommandoen gjøre det.
+
+Interessant og kanskje overraskende nok, er det i Monero **alltid** klart hva som er det neste som må skje når det kommer til multisig, bortsett i tilfeller av delvis signerte transkasjoner der du kan bestemme **hvilken** signerer å sende dem til, og i tilfeller av fullt signerte transaksjoner der du selv kan sende inn til nettverket eller sende dem til en annen signerer for at de skal sende det inn.
+
+Den spesielle kommandoformen `mms next sync` er for tilfeller der synkroniseringsdata venter som MMS-en i seg selv ikke kan behandle fordi den «tror» lommeboken er i en status som ikke trenger ny synkronisering – som kan være feil. Mer om dette kan du se i kapittelet *Feilsøking*.
+
+### mms sync
+
+    mms sync
+
+Starter manuelt en runde med tvangssynkronisering, dvs. hvis MMS-en er av den oppfatning at ingen utveksling av synkroniseringsdata er nødvendig. Mer om dette kan du finne i kapittelet *Feilsøking*.
+ 
+### mms transfer
+
+    mms transfer <transfer_command_arguments>
+
+Starter en overføring under MMS-ens kontroll, der forskjellen mellom den og standard `transfer`-kommandoen er at den resulterende, delvis signerte transaksjonen ikke blir skrevet til en fil som du må gjøre noe med selv, men at en melding som inneholder transaksjonen vil resulterer i det. Bruk `mms next` etter `mms transfer` for å be MMS-en om å faktisk behandle meldingen, noe som videre innebærer å bestemme hvilken signerer å sende den til for neste signatur, og å opprette enda en melding for dette.
+
+Argumentene til `mms transfer`-kommandoen er nøyaktig de samme som de av den standard `transfer`-kommandoen. Sjekk informasjonen om den kommandoen med `help transfer` for å lære alt om de forskjellige, mulige parameterene og parameterkombinasjonene.
+
+Mer at MMS-en generelt ikke bryr seg om adresser, underadresser og kontoer. Uavhengig av hva du spesifiserer, vil det alltid være en enkel, ny melding som inneholder den delvis signerte transaksjonen.
+
+Selv med MMS-en aktiv, kan du fremdeles bruke den standard `transfer`-kommandoen; da må du rett og slett håndtere transaksjonen på egenhånd. Prøv å bruke den riktige kommandovarianten; `transfer` vil ikke be om bekeftelse, enten du tiltenker å bruke den i stedet for `mms transfer`. Hvis du eksekverte `transfer`, men egentlig ønsket MMS-varianten, kan du ignorere den skriftlige transaksjonen og bare fortsette med `mms transfer`.

```

Comment:
typo: bekeftelse*

Reply:
Fixed. I used «planlegger» in stead of «tenker»